app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -32,6 +33,15 @@ def rhsm_organization_label
         self.organization.label
       end
 
+      def info_with_katello
+        info = info_without_katello
+        info['parameters']['kt_env'] = self.lifecycle_environment.try(:label) #deprecated
+        info['parameters']['kt_cv'] = self.content_view.try(:label) #deprecated
```

Comment:
I would change it so that it's own shown if they are set, but I wouldn't remove this information from the host parameters. 

It's useful because it becomes available as a global variable in Puppet manifests, and there are surely users who have built some logic around it.